.gitignore

```diff
@@ -23,4 +23,7 @@
 # Ignore env files.
 .ruby-version
 .ruby-gemset
-.env
\ No newline at end of file
+.env
+
+# Ignore idea files (used by RubyMine).
+.idea/
```

Comment:
If you use RubyMine as your editor of choice, it might be a good idea to add it permanently to your own, global gitignore.

Example here:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c